src/main/java/com/github/javafaker/Compass.java

```diff
@@ -0,0 +1,22 @@
+package com.github.javafaker;
+
+public class Compass {
+    private final Faker faker;
+
+    protected Compass(Faker faker) {
+        this.faker = faker;
+    }
+
+    public String directions() {

```

Comment:
be good to make that consistent with the string key in the yml file. Maybe go with the singular form, so "direction"

Reply:
Missed that. Fixed.